weblate/utils/markdown.py

```diff
@@ -24,7 +24,7 @@ def get_mention_users(text):
     )
 
 
-class WeblateHtmlRenderer(misaka.SaferHtmlRenderer):
+class WeblateHtmlRenderer(mistletoe.SaferHtmlRenderer):
```

Comment:
The API is different, it will need more work than just replacing library name...

Reply:
do I have to use HtmlRenderer which comes from mistletoe itself instead of using SaferHtmlRenderer?

Reply:
Yes, but it will probably need additional tweaking to behave same - it doesn't allow HTML to be used in the markup.